src/components/search/CustomerProfile.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+
+function CustomerProfile({ id }) {
```

Comment:
Align your code to use arrow function.  See this link for the benefits of using arrow function. https://www.freecodecamp.org/news/when-and-why-you-should-use-es6-arrow-functions-and-when-you-shouldnt-3d851d7f0b26/